cmd/root.go

```diff
@@ -110,7 +110,7 @@ func Execute(version, commit string) {
 
 func init() {
 	RootCmd.PersistentFlags().StringVarP(&gConfigFilePath, "config", "C", "", "Specify an alternate config file [env EXOSCALE_CONFIG]")
-	RootCmd.PersistentFlags().StringVarP(&gAccountName, "account", "A", "", "Account to use in config file [env EXOSCALE_ACCOUNT]")
+	RootCmd.PersistentFlags().StringVarP(&gAccountName, "accountt", "A", "", "Account to use in config file [env EXOSCALE_ACCOUNT]")
```

Comment:
WIP need some proposition

Reply:
we remove it and over an easy way to switch between them.

Reply:
--ACCOUNT can't be fine ?
```suggestion
	RootCmd.PersistentFlags().StringVarP(&gAccountName, "ACCOUNT", "A", "", "Account to use in config file [env EXOSCALE_ACCOUNT]")
```

Reply:
or longer? `--use-account` ?

Reply:
yes can be great !
```suggestion
	RootCmd.PersistentFlags().StringVarP(&gAccountName, "use-account", "A", "", "Account to use in config file [env EXOSCALE_ACCOUNT]")
```